Dockerfile

```diff
@@ -11,7 +11,9 @@ RUN apt-get update && apt-get install -y jq wget && \
 
 COPY . /app
 
-RUN pip install --upgrade pip && \
+RUN python3 -m venv venv && \
+    . venv/bin/activate && \
+    pip install --upgrade pip && \
     pip install --no-cache-dir -e .
```

Comment:
Isn't this PR for removing --no-cache-dir from pip install?

Reply:
yea we were just testing stuff out bc we're seeing unexpected behavior on Jenkins